src/tiledb/cloud/utilities/wheel.py

```diff
@@ -37,62 +35,27 @@ def upload_wheel(
       udflib-0.1-py3-none-any.whl
 
     :param wheel_path: path to the local wheel file
-    :param storage_uri: URI where the wheel filestore will be created or updated
-    :param wheel_name: name of registered wheel, defaults to a name based on the
-        wheel file
+    :param dest_uri: URI where the wheel filestore will be created or updated
     :param config: config dictionary, defaults to None
-    :param acn: TileDB Cloud access credentials name, defaults to None
-    :param namespace: TileDB Cloud namespace, defaults to the user's default namespace
-    :param register: register the wheel on TileDB Cloud, defaults to True
     """
 
-    # The wheel path may contain '+' characters, which are not allowed in TileDB URL
-    # encoded URIs and array names. Replace the '+' with '.' in `wheel_file`, which
-    # is used to create the storage URI and the array name. When installing the wheel,
-    # the original wheel file name will be recovered from metadata.
-    wheel_file = wheel_name or os.path.basename(wheel_path)
-    wheel_file = wheel_file.replace("+", ".")
-    array_uri = storage_uri.rstrip("/") + f"/{wheel_file}"
-
-    namespace = namespace or tiledb.cloud.user_profile().default_namespace_charged
-    tiledb_uri = f"tiledb://{namespace}/{wheel_file}"
+    # Replace '+' characters, which are not allowed in TileDB URL encoded URIs.
+    # When installing the wheel, the original wheel file name will be recovered
+    # from metadata.
+    dest_uri = dest_uri.replace("+", ".")
 
     with tiledb.scope_ctx(config):
         # Create the array if it doesn't exist
-        object_type = tiledb.object_type(array_uri)
+        object_type = tiledb.object_type(dest_uri)
         if object_type is None:
-            print(f"Creating wheel filestore - '{array_uri}'")
-            tiledb.Array.create(array_uri, tiledb.ArraySchema.from_file(wheel_path))
+            tiledb.Array.create(dest_uri, tiledb.ArraySchema.from_file(wheel_path))
+            print(f"Created filestore at '{dest_uri}'")
         elif object_type == "group":
```

Comment:
Won't `object_type` be an `array` if the wheel is already registered since the wheel is registered as an array?